### PR TITLE
Reviewed GeoServer Quickstart

### PR DESCRIPTION
**Pull request by flicstar**
Reviewed as part of the QuickStart project during Google Season of Docs 2019. The aim is consistency across all OSGeoLive QuickStarts.
I added numbered steps and changed heading style and wording. Also modified the What Next? section. @fgdrf Please action comments and suggestions you agree with, and ignore those you don't.

Partner trac ticket link: https://trac.osgeo.org/osgeolive/ticket/2210

See this page for reference: https://trac.osgeo.org/osgeolive/wiki/How%20to%20document%20the%20quickstart%20file

### CODE REVIEW

File: doc/quickstart/geoserver_quickstart.rst [hunk 1]
```diff
@@ -29,35 +29,32 @@ This Quick Start describes how to:
...
+   :local:
   
-Start |GS|
-================================================================================
+Start Geoserver
+===============
 
-#. Select the |osgeolive-appmenupath-geoserver| in the menu.
-#. The application will take a few moments to start up and will open a web page at http://localhost:8082/geoserver/web 
+#. From the Start menu, select :menuselection:`Geospatial --> Web Services --> GeoServer --> Start GeoServer`. The application will take a few moments to start up and will open a web page at http://localhost:8082/geoserver/web
```
Comment (flicstar) [inline doc/quickstart/geoserver_quickstart.rst]: Note, I couldn't get the service to start on either 13.0 or version 12.0. I reviewed the doc without being able to truly run through the procedure.

File: doc/quickstart/geoserver_quickstart.rst [hunk 2]
```diff
@@ -29,35 +29,32 @@ This Quick Start describes how to:
...
 
 .. contents:: Contents
+   :local:
   
-Start |GS|
-================================================================================
+Start Geoserver
+===============
 
-#. Select the |osgeolive-appmenupath-geoserver| in the menu.
```
Comment (fgdrf) [inline doc/quickstart/geoserver_quickstart.rst]: Replacement for |osgeolive-appmenupath-geoserver| as :menuselection:`Geospatial --> Web Services --> GeoServer --> Start GeoServer` has been introduced while shell-scripts can change installation path so we do not have to update docs.

File: doc/quickstart/geoserver_quickstart.rst [hunk 3]
```diff
@@ -267,31 +232,23 @@ Now click :guilabel:`Save` and go to the Layer Preview page to check that it loo
...
-================================================================================
-
-A large variety of clients exist to make use of the `WMS <http://www.opengeospatial.org/standards/wms>`__ layers you are serving
-from |GS|. This is a list of just some of them 
-
-    * :doc:`uDig <../overview/udig_overview>`
-
-    * :doc:`OpenLayers <../overview/openlayers_overview>`
-
-    * :doc:`MapBender <../overview/mapbender_overview>`
```
Comment (fgdrf) [inline doc/quickstart/geoserver_quickstart.rst]: Any reason why this section has been removed. From my perspective it cross-reference other packages distrubuted by OSGeo live and its worth for users to see how these might work togeher

File: doc/quickstart/geoserver_quickstart.rst (hunk 2 above)
Comment (flicstar) [reply doc/quickstart/geoserver_quickstart.rst]: Okay, I will revert.

File: doc/quickstart/geoserver_quickstart.rst (hunk 3 above)
Comment (flicstar) [reply doc/quickstart/geoserver_quickstart.rst]: Okay, I will re-instate. It seemed out of place but now that you've explained it, it makes sense.

File: doc/quickstart/geoserver_quickstart.rst (hunk 1 above)
Comment (camerons) [reply doc/quickstart/geoserver_quickstart.rst]: @fgdrf @flicstar has anyone managed to get geoserver to start on OSGeoLive? We might have great docs now, but they are not very helpful if they are explaining an application which doesn't work.
@fgdrf are you be in a position to download osgeolive 13.0 and see  if there is a problem we need fixing? We might need to raise an issue against the OSGeoLive software, or GeoServer software.

Comment (kalxas) [reply doc/quickstart/geoserver_quickstart.rst]: GeoServer works fine both on 12.0 and 13.0.
It could be a matter of RAM allocation to the VM.

Comment (camerons) [reply doc/quickstart/geoserver_quickstart.rst]: Thanks @kalxas, based on your comment, I will consider geoserver quickstart good for merging.

**Pull request merged**